lib/faker/pizza.ex

```diff
@@ -0,0 +1,162 @@
+defmodule Faker.Pizza do
+  import Faker, only: [sampler: 2]
+
+  @moduledoc """
+  Functions for generating Pizza related data in English
+  """
+
+  @doc """
+  Returns a list with a random number of pizzas (in between the specified
+  range)
+  If no range is specified it defaults to 2..5
+  """
```

Comment:
If I remember correctly, this documentation will be overridden by next one because function have the same arity, it's need to be rephrased and grouped into one.